scout/server/blueprints/phenotypes/views.py

```diff
@@ -28,3 +28,16 @@ def api_hpo_terms():
 
     data = controllers.hpo_terms(store=store)
     return jsonify(data)
+
+
+@hpo_bp.route("/api/v1/phenotypes/search/<search_string>", methods=["GET"])
```

Comment:
Where do you see in the react page that the address will become like this?

Reply:
I mean, shouldn't this be working with @elevu's example phenotype page?

Reply:
I dont think so yet. It wasn't there to call before, and I used endpoint request syntax instead of a get url '?' argument.

Reply:
The route is new, and can be seen working by typing it into a browser (compare picture above). 😊

Reply:
I assumed the page was a work in progress, but can have a look there as well. There were som DOM container errors as well that should not be related to a search returning results or not.

Reply:
Yes I noticed the DOM errors. I am asking because I don't know if @elevu and @moedarrah have a specific idea in mind on how to handle the phenotype search in their phenotypes page. Might also be that they retrieve all phenotypes and then filter on the fly using some js code instead of this type of search?

Reply:
Nah, then there would be no need for a new endpoint. The old one would have been sufficient if the js did some matching. There is also evident use if a search url, but I dont quite yet understand how it is supposed to switch between mock and live stage data. There are some hardcoded server urls there that I suppose are for testing:
https://github.com/Clinical-Genomics/scout-react/blob/2209e9800813c00d38a3fab72d80a55147f74d2d/src/services/ScoutApi.ts#L73

Reply:
But I guess providing a query option is always good to have, I am not really concerned about the query. But it would be nice to see that the CORS fix actually work with the react app before merging..

Reply:
Good point. I'll see what I can do: if I read that correctly, the setPhenotypes() function is still missing? 
The DOM errors are by the way unrelated - they appear on any page with the top menu, likely just a missing div id tag somewhere.

Reply:
For the record, it was there, from a little js hook `useState`, but the search function had been temporarily repurposed for other testing.

Reply:
we asked for the Search endpoint because GET /phenotypes returns thousands of objects, it is too slow even on the current Scout.
This new endpoint works perfectly on the React app, thank you!

Reply:
https://scout-stage.web.app/cust000/phenotypes some change is needed on the UI but the table with phenotypes works!

Reply:
Nice, we can merge this PR then!